### PR TITLE
LDP-16508 | master

### PR DESCRIPTION
**Pull request by marcosapmf**
Jira ticket: https://issues.liferay.com/browse/LDP-16508

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

File: test.properties [hunk 1]
```diff
@@ -3436,6 +3439,8 @@
     # Analytics Cloud
     #
 
+    playwright.test.project[playwright-js-tomcat90-mysql57-jdk8][analytics-cloud]=analytics-settings
```
Comment (CalumR23) [inline test.properties:3442]: playwright.test.project[playwright-js-tomcat90-mysql57-jdk8][analytics-cloud]=analytics-settings,${osbfaroProjectName}

**Comment by ViniLopes20** [comment]
Hey @CalumR23, after the changes in this PR, then `ci:test:analytics-cloud` will run all Poshi and Playwright tests, right? The Playwright results will be imported into which AC project execution routine?

File: test.properties [hunk 2]
```diff
@@ -3448,7 +3453,7 @@
     test.batch.dist.app.servers[analytics-cloud]=tomcat
 
     test.batch.names[analytics-cloud]=\
-        functional-tomcat90-mysql57-jdk8
+        playwright-js-tomcat90-mysql57-jdk8
```
Comment (CalumR23) [inline test.properties:3456]: Hey @ViniLopes20 for this change, keep this line and add back in the old line of functional-tomcat90 for both Poshi and Playwright tests

**Comment by CalumR23** [comment]
@marcosapmf Could you add the osb faro web project to the `test.properties` file and re-run? It looks like it didnt run the AC tests in the previous run but the batch shows it was grabbing the correct tests.

File: test.properties (hunk 2 above)
Comment (ViniLopes20) [reply test.properties:3457]: Done, thanks @CalumR23

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

**Comment by marcosapmf** [comment]
ci:stop

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:stable

**Comment by marcosapmf** [comment]
ci:test:stable

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:stable

**Comment by marcosapmf** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
Send again here: https://github.com/liferay-ac/liferay-portal/pull/1183

**Comment by ViniLopes20** [comment]
ci:stop

**Pull request closed without merging**